spec/models/action_spec.rb

```diff
@@ -3,10 +3,22 @@
 describe Action do
   let(:page) { create :page }
 
-  it 'increases the action_count after creation' do
-    create :action, page_id: page.id
-    expect(page.reload.action_count).to eq 1
-    create :action, page_id: page.id
-    expect(page.reload.action_count).to eq 2
+  describe 'on create' do
+    subject{ create(:action, page_id: page.id) }
+
+    describe 'counter_cache on page' do
+      it 'increases the action_count after creation' do
+        expect { subject}.to change{ page.reload.action_count }.from(0).to(1)
+      end
+
+      it 'does not stamp updated_at' do
+        expect { subject }.not_to change{ page.reload.updated_at }
+      end
+
+      it 'does not change cache_key' do
+        expect { subject }.not_to change{ page.reload.cache_key }
+      end
+    end
```

Comment:
@NealJMD good catch. I've changed this slightly - it now uses rails' `counter_cache` option, which means less code for us to maintain. I've also changed the rspec matchers for the specs. We should have one assertion per test.